.bcr/metadata.template.json

```diff
@@ -1,6 +1,21 @@
 {
   "homepage": "https://github.com/bazelbuild/rules_proto",
-  "maintainers": [],
+  "maintainers": [
+    {
+      "github": "comius",
```

Comment:
I think @fowles should be listed as a maintainer. I'm just helping in the transition period.

@fowles is that ok with you?

Reply:
can we start with @googleberg and @zhangskz ?